README.md

```diff
@@ -38,6 +38,9 @@ Building
 
 For platforms with Autotools, the generic way to build and install it is:
 
+
+    sudo apt install libcrypto++-dev libsqlite3-dev libc-ares-dev libcurl4-openssl-dev zlib1g-dev libfreeimage-dev libreadline-dev libsodium-dev
```

Comment:
The build dependencies are already listed further down under "Platform Dependencies".

Reply:
Yes, it's my personal change to remind myself, I just want to copy&paste them when rebuilding the MegaSDK in a new Ubuntu env. So just ignore this. Thanks!